server/bastionlab_polars/src/lib.rs

```diff
@@ -531,4 +547,69 @@ impl PolarsService for BastionLabPolars {
         );
         Ok(Response::new(Empty {}))
     }
+    async fn split(
```

Comment:
same train_test_split comment

Reply:
Well, as I said, the features are inter-related and I cannot use RemoteTokenizers without the data-conversion feature.
And splitting cannot be implemented without RemoteArray's (data-conversion) feature.